bundles/net.tourbook.export/src/net/tourbook/export/DialogExportTour.java

```diff
@@ -1470,14 +1470,15 @@ private void setFileName() {
 
       // search for the first tour
       TourData minTourData = _tourDataList.get(0);
-      final long minTourMillis = 0;
+      long minTourMillis = minTourData.getTourStartTime().toInstant().toEpochMilli();
```

Comment:
Are you shure that `.get(0)` is not `null`?

There is also this method `TourData.getTourStartTimeMS()`

Reply:
Yes, because the export is not launched if it doesn't contain any tours

See [here](https://github.com/mytourbook/mytourbook/blob/d5982fbcb2c41f1c7437d1a79bc361511498f48d/bundles/net.tourbook/src/net/tourbook/extension/export/ActionExport.java#L77)